src/tiles/models/tilesManager.ts

```diff
@@ -48,6 +48,11 @@ export class TilesManager {
     }
   }
 
+  public async addTilesToQueue(tiles: Tile[]): Promise<void> {
+    const tileArr = tiles.map((tile) => ({ name: this.tilesQueueName, data: tile }));
```

Comment:
rename to something that indicates that it is in fact jobs

Reply:
its not

Reply:
renamed